api/v1/coupons.js

```diff
@@ -27,6 +27,18 @@ var getCouponCodesByCouponID = function(req, res, next) {
 };
 exports.getCouponCodesByCouponID = getCouponCodesByCouponID;
 
+var createCouponForNewUser = function (req, res, next) {
+  //No need for adminAuth
+  coupon.createCouponWithDefaultRulesForNewUser(req.userName, req.couponCode).then(function (coupon) {
+      res.statusCode = 201;
+      res.send(coupon);
+  }, function (err) {
+      err.api = true;
+      next(err);
+  });
+};
+exports.createCouponForNewUser = createCouponForNewUser;
```

Comment:
why do we need a create coupon for new user? This will create a coupon not a user.